test/unit/elsa/group/acknowledger_test.exs

```diff
@@ -1,8 +1,8 @@
 defmodule Elsa.Group.AcknowledgerTest do
   use ExUnit.Case
 
-  alias Elsa.{Registry, Supervisor}
   alias Elsa.Group.Acknowledger
+  alias Elsa.{Registry, Supervisor}
```

Comment:
Should we be aliasing over things like https://hexdocs.pm/elixir/1.18.3/Supervisor.html ? that seems really confusing for reading this code later.

Reply:
Yeah that's horrible, I'm gonna change it